implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -164,7 +164,13 @@ public OpenApiAnnotationScanner(OpenApiConfig config, IndexView index) {
      */
     public OpenApiAnnotationScanner(OpenApiConfig config, IndexView index, List<AnnotationScannerExtension> extensions) {
         this.config = config;
-        this.index = index;
+
+        if (index instanceof FilteredIndexView) {
+            this.index = index;
+        } else {
+            this.index = new FilteredIndexView(index, config);
+        }
```

Comment:
Nice.  Make sure it's always filtered.  +1

Reply:
Also I'm excited to have used multi-line commenting for the first time.  Way to go, GitHub. :)

Reply:
> 
> 
> Also I'm excited to have used multi-line commenting for the first time. Way to go, GitHub. :)

It's pretty nice - helps with context!